src/web/services/table-comparator.service.ts

```diff
@@ -134,6 +168,8 @@ export class TableComparatorService {
       case SortBy.MSQ_WEIGHT_TOTAL:
       case SortBy.MSQ_WEIGHT_AVERAGE:
         return this.compareNumbers(strA, strB, order);
+      case SortBy.LOG_DATE:
+        return this.compareChronologically(strA, strB, order, 'ddd, DD MMM YYYY hh:mm:ss A');
```

Comment:
This looks like a magic string, I recognise that this change to add a param here was made in response to a previous comment, but it seems odd to leave it as-is.

I wonder if we could do away with this `'ddd, DD MMM YYYY hh:mm:ss A'` string completely? After all, running something like `moment('Mon, 21 Jun, 2021 04:25:26 PM')` still parses it correctly without needing to supply the exact format.

Reply:
I have kept the date time formats as magic strings, as making it a class constant seems to result in circular dependency.

![image](https://user-images.githubusercontent.com/55139982/134812376-a0e532cc-8fcc-4c8f-a5ec-52b826b1130b.png)

Reply:
I've updated this part based on your comments. The circular dependency has been resolved as well by doing so.

Reply:
ah thanks!